core/animation.cpp

```diff
@@ -7,7 +7,7 @@
  * repository for more details.
  **********************************************************************/
 
-#include <climits>
+#include <limits>
```

Comment:
The compiler was complaining that it couldn't find the `std::numeric_limits` - this fixed it